.eslintrc.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "./node_modules/gts/"
```

I usually also set these (apologies for single quote, I copied it from another project):
```suggestion
  "extends": "./node_modules/gts/"
  "rules": {
        "object-curly-spacing": ["error", "always"],
        "require-jsdoc": "off",
        'valid-jsdoc': 'off',
        'prettier/prettier': 'error',
        '@typescript-eslint/no-unused-vars': 'warn',
        '@typescript-eslint/no-floating-promises': 'error',
    },
```

This is my previous project settings, feel free to copy if that works: [uchen-ml/github-sampler@.eslintrc.js](https://github.com/uchen-ml/github-sampler/blob/main/.eslintrc.js)